packages/aragon-cli/test/start/start.test.js

```diff
@@ -1,32 +0,0 @@
-import test from 'ava'
-import execa from 'execa'
-
```

Comment:
If this test is removed, then are we testing the start command anywhere else?

Reply:
Yep, I move it to the boilerplates